soccer/planning/RRTPlanner.cpp

```diff
@@ -157,6 +158,7 @@ std::unique_ptr<InterpolatedPath> RRTPlanner::generateRRTPath(
         // Check if Planning or optimization failed
         if (points.size() < 2) {
             debugLog("RRTPlanning Failed");
+            exit(1);
```

Comment:
This seems a little aggressive. I can still get these error messages in the logs from time to time, we don't want to quit soccer when they happen...